libraries/rush-lib/src/utilities/test/npmrcUtilities.test.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { trimNpmrcFileLines } from '../npmrcUtilities';
+
+describe('npmrcUtilities', () => {
+  it(trimNpmrcFileLines.name, () => {
```

Comment:
Add some cases without env vars and with malformed expansion expressions. Also add some multiline cases.